sfs/mono/source.py

```diff
@@ -79,6 +79,54 @@ def point_velocity(omega, x0, n0, grid, c=None):
     return util.XyzComponents([v * o / r for o in offset])
 
 
+def point_dipol(omega, x0, n0, grid, c=None):
+    """Point source with dipole characteristics.
+
+    Parameters
+    ----------
+    omega : float
+        Frequency of source.
+    x0 : (3,) array_like
+        Position of source.
+    n0 : (3,) array_like
+        Normal vector (direction) of dipole.
+    grid : triple of numpy.ndarray
+        The grid that is used for the sound field calculations.
+        See :func:`sfs.util.xyz_grid`.
+    c : float, optional
+        Speed of sound.
+
+    Returns
+    -------
+    numpy.ndarray
+        Sound pressure at positions given by `grid`.
+    ::
```

Comment:
Should I move it behind or before the Examples section. If you say "first unnamed section" it sounds like it should go to the very end, which is fine as in my opnion the equation should be directly above the actual code.
I'm asking, because in the case of the point source it looks like this at the moment:
```python
def point(omega, x0, n0, grid, c=None):
    """Point source.

    ::

                      1  e^(-j w/c |x-x0|)
        G(x-x0, w) = --- -----------------
                     4pi      |x-x0|

    Examples
    --------
    .. plot::
        :context: close-figs

        p = sfs.mono.source.point(omega, x0, None, grid)
        sfs.plot.soundfield(p, grid)
        plt.title("Point Source at {} m".format(x0))
```